src/core/components/layout-utils/drop-down.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { PureComponent } from "react"
+import PropTypes from "prop-types"
+import cx from "classnames"
+
+export default class DropDown extends PureComponent {
+
+  static propTypes = {
+    id: PropTypes.string,
+    label: PropTypes.string,
+    displayLabel: PropTypes.string,
+    mod: PropTypes.string,
+    disbaled: PropTypes.bool,
+    children: PropTypes.node.isRequired
+  }
+
+  constructor(props) {
+    super(props)
```

Comment:
FYI you don't need to pass `props` if you're not using them within the constructor